.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-FORCE_COLOR=1 pnpm -- lerna run --concurrency 1 --stream precommit --since HEAD --exclude-dependents
+FORCE_COLOR=1 pnpm -r --filter "[HEAD]" precommit
```

Comment:
Maybe we still need `--workspace-concurrency=1`?

Reply:
no need i think, i remember adding `concurrency` was caused by a bug of lerna. let's see.